src/zenml/zen_stores/migrations/README.md

```diff
@@ -0,0 +1,51 @@
+# How to create a migration
+
+## How alembic is works
+
+src/zenml/alembic.ini is used for the configuration of alembic in general. The
```

Comment:
I think the `alembic.ini` file is missing in this PR? Also would it be possible to move that file into `src/zenml/zen_stores/migrations` or does it need to stay at the top level?

Reply:
I actually moved it out of the python package, into the repository root. It won't be part of the python package (still debatable if it should be).